core/iwasm/common/wasm_runtime_common.h

```diff
@@ -264,7 +271,67 @@ STORE_U16(void *addr, uint16_t value)
     ((uint8_t *)(addr))[0] = u.u8[0];
     ((uint8_t *)(addr))[1] = u.u8[1];
 }
+
```

Comment:
IIUC, the key assumption for both, `STORE_V128` and `LOAD_V128`, functions to work is that WAMR runs on aligned addresses. If I remember right, WAMR operates in unaligned mode by default, on x86 and x86-64 at least(?). 

Please correct me if it is no longer the case. If yes, I would suggest adding a `WASM_ENABLE_xxx` or just using `WASM_ENABLE_SIMDE` to differentiate it from the others.